routing/router.go

```diff
@@ -1020,6 +1020,7 @@ func (r *ChannelRouter) SendPayment(payment *LightningPayment) ([32]byte, *Route
 	// target payment using the multi-hop route. We'll try each route
 	// serially until either once succeeds, or we've exhausted our set of
 	// available paths.
+Routes:
```

Comment:
Minor nit: all other named labels in the codebase start with a lower case letter.